README.md

```diff
@@ -1,6 +1,6 @@
 # Doctor Mark
 
-An Automatic Page Generation Tool http://jxnblk.com/doctor-mark
+An Automatic Page Generation Tool <http://jxnblk.com/doctor-mark>
```

Comment:
I was hoping this would cause the URL to be linked on https://www.npmjs.com/package/doctor-mark